Documentation/Architecture/INetworkingBase.md

```diff
@@ -0,0 +1,44 @@
+# NetworkingBase<T> Class
+
+## Properties
+
+### NetworkingMode
+
+This is the selected type of networking that will be used.
+It is selected from the following NetworkingType Enum: [NetworkingType](./NetworkingType.md)
+
+Is set to Photon by default.
+
```

Comment:
This should not be in the base, but rather provided by the concrete class

Reply:
`IMixedRealityNetworkSystem`

Reply:
Also, why are we going to set it to photon by default?

Shouldn't we have it set to either uNET (as it's built into the editor) or our planned WebRTC default?